pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/codestyle/CommentDefaultAccessModifierRule.java

```diff
@@ -41,6 +41,14 @@
 
     public CommentDefaultAccessModifierRule() {
         definePropertyDescriptor(REGEX_DESCRIPTOR);
+        defaultSuppressionAnnotations();
```

Comment:
there is no need to call this method here

Reply:
Yes, this is called already by the super class, when it is defining the property:

https://github.com/pmd/pmd/blob/7edaf78e4a61a7e49d8ca67e21fbed4e01fb9ffd/pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/AbstractIgnoredAnnotationRule.java#L18